clients/js/test/legacy/sell.test.ts

```diff
@@ -1510,3 +2480,304 @@ test('token pool with makerBroker set requires passing the account in & fails w/
   // Should fail with a missing makerBroker error.
   await expectCustomError(t, promise, TENSOR_AMM_ERROR__WRONG_MAKER_BROKER);
 });
+
+test('alternate deposits & sells', async (t) => {
```

Comment:
what program functionality is this test supposed to assert?

Reply:
This is ported over from TSWAP, it's just doing a bunch of deposits and sells to make sure pools don't have edge cases for a lot of actions.